crates/rpc-types/src/eth/transaction/request.rs

```diff
@@ -14,18 +14,23 @@ pub struct TransactionRequest {
     /// The destination address of the transaction.
     pub to: Option<Address>,
     /// The legacy gas price.
-    #[serde(default)]
+    #[serde(default, with = "alloy_serde::num::u128_hex_or_decimal_opt")]
     pub gas_price: Option<u128>,
     /// The max base fee per gas the sender is willing to pay.
-    #[serde(default)]
+    #[serde(default, with = "alloy_serde::num::u128_hex_or_decimal_opt")]
     pub max_fee_per_gas: Option<u128>,
     /// The max priority fee per gas the sender is willing to pay, also called the miner tip.
-    #[serde(default)]
+    #[serde(default, with = "alloy_serde::num::u128_hex_or_decimal_opt")]
     pub max_priority_fee_per_gas: Option<u128>,
```

Comment:
`skip_serializing_if`

Reply:
isnt this handled by the serde_with?

Reply:
it will be serialized as null by default